model/jpa/src/main/java/org/keycloak/connections/jpa/updater/JpaUpdaterProvider.java

```diff
@@ -49,23 +49,26 @@
      * Updates the Keycloak database
      * @param connection DB connection
      * @param defaultSchema DB connection
+     * @param verifyAndRunMasterChangelog if master changelog should be verified and run 
      */
-    void update(Connection connection, String defaultSchema);
+    void update(Connection connection, String defaultSchema, boolean verifyAndRunMasterChangelog);
```

Comment:
Why the need to introduce verifyAndRunMasterChangelog? What does it do?

Reply:
We tell the updater whether or not we should validate/execute the master changelog.

Running validation and checking the "unrun" changesets during startup is one of the main parts that affect startup, as well as footprint. This is especially true if the database is already initialized.